### PR TITLE
Add my draft of places to do after 'git add' (filled avtomatikally)

### LINKED ISSUE #7: sergey179 recommendations for things to do
Things to do

- [ ] Places to eat
- [x] Things to do

@affrae

### PR DESCRIPTION
**Pull request by sergey179**
This closes #7 

adding a file with commendations for Kto?

@nikosandronikos  what should I add?

### CODE REVIEW

**Comment by andrewdorrell** [comment]
Nice rec's!

File: sergey-somefile.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+things to do
+bla
+places to go:wq
```
Comment (andrewdorrell) [inline sergey-somefile.md]: Looks like a stray VI command in there!

**Review by andrewdorrell — CHANGES_REQUESTED** [review]
obviously still needs some work but the direction is good

File: sergey-somefile.md (hunk 1 above)
Comment (sergey179) [reply sergey-somefile.md]: Did I type go:wq?

Comment (sergey179) [reply sergey-somefile.md]: Did I type go:wq?

File: sergey-somefile.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+things to do
```
Comment (sergey179) [inline sergey-somefile.md:1]: things => Things

**Pull request merged**